### DISCUSSION TITLE
Migrate from MFC to Qt 6.x or WinUI v3.x

### DISCUSSION THREAD

**Discussion by ansani** [ansani/Shareaza#20] [Ideas]
While analyzing the code to fix some issues, I figured some patterns are a bit old. Maybe a good idea to migrate to a new UI framework (QT or WinUI). What do you think?

**Comment by cardpuncher** [comment]
Hi,

I remember from the old days that the dependence on MFC was one of the main reasons for which Shareaza couldn't run on anything else than Windows, so maybe migrating to Qt could make that easier too. There was even a project aiming  to port Shareaza to Qt, Panthera, but it didn't succeed (TorrentFreak has articles about that).

But that would be a huge undertaking.

**Comment by GreenLunar** [comment]
@cardpuncher [Panthera](http://torrentfreak.com/shareaza-team-fight-back-with-panthera-project-080818/) and [Quazza](https://sourceforge.net/projects/quazaa/) too.

I suggest to migrate to Qt in order to make Shareaza to work on *NIX platforms too, and thereby growing the active members, developers and contributors to this project.

**Reply by ansani** [reply]
Yes, I started a fork but it's still in pre-alpha :/

**Reply by GreenLunar** [reply]
Please do let me know when you need Linux testers. I'm using ArchLinux and be delighted to provide any feedback.
